Packs/rasterize/Integrations/rasterize/rasterize.py

```diff
@@ -235,6 +236,26 @@ def network_request_will_be_sent(self, documentURL, **kwargs):
         '''Triggered when a request is sent by the browser, catches mailto URLs.'''
         demisto.debug(f'PychromeEventHandler.network_request_will_be_sent, {documentURL=}')
         self.is_mailto = documentURL.lower().startswith('mailto:')
+
+        request_url = kwargs.get('request', {}).get('url', '')
+
+        if any(value in request_url for value in BLOCKED_URLS):
+            self.tab.Fetch.enable()
+            demisto.info('Issuing of requestPaused events enabled.')
+
+
+
+    def handle_request_paused(self, **kwargs):
+        request_id = kwargs.get("requestId")
+        request_url = kwargs.get("request").get("url")
+
+        if any(value in request_url for value in BLOCKED_URLS):
```

Comment:
Need to check that it's a `redirect` request that you're blocking.
e.g. we don't want to block images.